cmd/homi/docker/compose/homi.go

```diff
@@ -39,19 +39,25 @@ type Homi struct {
 }
 
 func New(ipPrefix string, number int, secret string, addresses, nodeKeys []string,
```

Comment:
Just minor comment.
How about replacing  `number` -> `numPNs` or remove it? :)

Reply:
The `number` is for CN. I'd like to do refactoring at the next PR.